ads/google/csa.js

```diff
@@ -212,15 +212,24 @@ function requestCsaAds(global, data, afsP, afsA, afshP, afshA) {
  */
 function getAdType(data) {
   if (data['afsPageOptions'] != null && data['afshPageOptions'] == null) {
+    validateData(data, ['afsPageOptions', 'afsAdblockOptions'], []);
```

Comment:
Yes,  it's to check that only these two are present.  They are both required and there shouldn't be any other attributes present if data['afshPageOptions'] == null (in this case).